lwk/lwkwallet.go

```diff
@@ -18,10 +18,12 @@ import (
 type Satoshi = uint64
 
 // SatPerKVByte represents a fee rate in sat/kb.
-type SatPerKVByte = uint64
+type SatPerKVByte = float64
 
 const (
-	minimumSatPerByte = 200
+	minimumSatPerByte SatPerKVByte = 0.1
```

Comment:
Named `minimumFee` because it is understandable from the type definition.

[`7045055` (#288)](https://github.com/ElementsProject/peerswap/pull/288/commits/7045055a00ffb06466bf9312fc23cb1a41e3e364)